### DISCUSSION TITLE
Quarto not displaying on other computers correctly

### DISCUSSION THREAD

**Discussion by austin-putz** [quarto-dev/quarto-cli#2933] [Q&A]
Hi all, 

I render my Quarto doc on my Work PC. I then send them out via teams or email. However, when I do that it removes the plots and displays in a very poor manner. However, on the computer I rendered it on, looks perfect. 

Any ideas for me? Very frustrating. 

Thanks,
Austin
![Screen Shot 2022-10-19 at 11 11 05 AM](https://user-images.githubusercontent.com/12675488/196746452-bc48b282-db2c-4631-9365-dba79e4ca944.png)

**Comment by dragonstyle** [comment]
It's likely that you're just sending a standalone HTML file and not the files it references (in the `_files` folder). 

You can use `embed-resources: true` in your document to place these references inline in the HTML.

**Reply by austin-putz** [reply]
Thank you but I'm still struggling. I've tried several times, rendered, sent and still the same. Where exactly do I place it? It's not coming up when I hit tab in RStudio anywhere. But the deprecated 'standalone' comes up, but still doesn't work...

**Reply by austin-putz** [reply]
Got it, 'self-contained' before format: worked for me. Thanks.

**Reply by cscheid** [reply]
Are you using the latest 1.2 prerelease of quarto? That is the version that supports `embed-resources: true`.

**Reply by austin-putz** [reply]
I don't think I am, I forgot to update my work computer, only my personal computer. I'll download and try again.

**Reply by alexpaterson31** [reply]
> It's likely that you're just sending a standalone HTML file and not the files it references (in the `_files` folder).
> 
> You can use `embed-resources: true` in your document to place these references inline in the HTML.

Thank you this was massively helpful!
